webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -803,11 +849,13 @@ private static TableAndHighlighting createTable(PdfWriter docWriter, Document do
             // Again, I have no idea where this number is coming from. I'm chalking it up to
             // unaccounted for margins.
             int WIDTH_MARGINS = 40;
-            Rectangle availableArea = new Rectangle(scrambleColumnWidth - WIDTH_MARGINS,
+            Rectangle availableArea = new Rectangle(scrambleColumnWidth - 2*SCRAMBLE_PADDING_HORIZONTAL/* - WIDTH_MARGINS*/,
                     availableScrambleHeight - HEIGHT_MARGINS);
+            System.out.println("Normal..." + scrambler.getShortName());
```

Comment:
Please no print statements in the final version of tnoodle. If you really need something, there's a logging system that lets us turn these things on and off as needed without recompiling tnoodle.